telemetry/event_counters.go

```diff
@@ -12,7 +12,12 @@ import (
 	"github.com/hashicorp/go-metrics"
 )
 
-const eventTypeMetricKey = "event_type"
+const (
+	eventTypeMetricKey = "event_type"
+	// TODO_INVESTIGATE(@okdas): Using eventTypeMetricKey for counter and gauge
+	// doesn't seem to work. Add "_gauge" postfix to display it in the meantime.
+	eventTypeMetricKeyGauge = "event_type_gauge"
```

Comment:
I'm a bit confused why we make `relay_mining_difficulty` and `relay_ema` a label. To me personally, it would make more sense if they would actually be the name of the metrics. Ideally, also namespaced from other, non-pocket specific metrics. For example:

`event_type_gauge{service_id="anvil",type="relay_ema"} 36` -> `pocket_relay_ema{service_id="anvil"} 36`
`event_type_gauge{service_id="anvil",type="relay_mining_difficulty"} 0` ->  `pocket_relay_mining_difficulty{service_id="anvil"} 0`

I understand the data might be coming from an event, but from monitoring/observability perspective we work with the data that the process exposes and have now knowledge of how the data is exposed.

Similarly, I'd rename `event_type` to `pocket_event_type`.

If that makes sense, I can take care of the changes on that branch. Just let me know @red-0ne

Reply:
I think you are right, but for consistency we'd have to replace all the `event_type`s with their corresponding events too.

I'll add a `TODO_DECIDE`. Tell me if that's OK with you.

Reply:
@red-0ne sure, let's do that! Thank you. I approved the PR - please merge at your convenience.